core/app/models/spree/shipping_method_stock_location.rb

```diff
@@ -0,0 +1,6 @@
+class Spree::ShippingMethodStockLocation < ActiveRecord::Base
```

Comment:
This should probably become nested. That way people know that `Spree` is a module... Makes code investigation for newcomers easier.

Also, it should extend from `Spree::Base`, not `ActiveRecord::Base`. This way we get that `spree_` table prefix.